type.proto

```diff
@@ -261,6 +262,15 @@ message Line {
     optional string text_color = 24;
 }
 
+message Polygon {
+    optional MultiLineString multi_lines = 1;
```

Comment:
I think you should split the multilinestring in:
* a linestring outer
* a list of linestring innner